packages/projects-docs/pages/tutorial/getting-started-with-docker.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Getting started with Docker and NixOS
-description: Using docker and docker-compose with CodeSandbox Projects
+description: Using docker and docker-compose with CodeSandbox.
 ---
 
 import Callout from 'nextra-theme-docs/callout'
 
 # Getting started with Docker and NixOS
 
-As an experimental feature, Docker and Nix are available within Projects. You can run and build images to run containers inside your project by opening a terminal and running `docker`. If you have a `docker-compose` file in your root folder you have to run only `docker-compose up` and that's all, all your services will run within Projects.
+Docker and Nix are available within CodeSandbox Repositories as an experimental feature. You can run and build images to run containers inside your project by opening a terminal and running `docker`. If you have a `docker-compose` file in your root folder, you only have to run `docker-compose up` - all your services will run within CodeSandbox Repositories.
```

Comment:
I would replace `-` with `then`:
"you only have to run `docker-compose up`. Then all your services will run within CodeSandbox Repositories.